### DISCUSSION TITLE
Workflow Issues After Upgrading to Visidata 3.1.1: Seeking Solutions

### DISCUSSION THREAD

**Discussion by DmitryAndreyeff** [saulpw/visidata#2628] [Q&A]
I frequently use Visidata to work on CSV reports. I have a VD script that opens multiple views of a file, primarily pivot tables, allowing me to quickly review the data and make changes as needed.

After upgrading to version 3, I noticed that my workflow has significantly worsened. Loading files and creating pivot tables now takes much longer—about 10 seconds compared to a split second previously. Additionally, the screen is cluttered with messages that weren’t present in the earlier version. On top of that, there’s now a somewhat annoying large green message box on the SheetsSheet, which I use often.

Is there a way to achieve a similar experience to what I had in version 2? For now, I’ve reverted to the previous version.

**Comment by anjakefala** [comment]
Could you try some things for Dmitry?

Could you try setting `options.disp_expert` to 5? And try setting `options.disp_sidebar` to False? Let me know if any of those improve your experience/performance.

In a live-environment `Ctrl+G` closes the sidebar if the help sidebar is popped up.

**Reply by DmitryAndreyeff** [reply]
Hi @anjakefala , I added these params in the command line like this `vd --disp_expert=5 --disp_sidebar=false -p myfile.vdj`, It did not solve my problem, I did not notice any change.

**Comment by midichef** [comment]
If you open an Issue and fill out the template there with more details, we can take a look to find out why loading and making pivot tables is slower for you. Some items in that template that are particularly helpful are: exact visidata versions (from `vd -V`), sample data files to reproduce the problem, details of OS version, Python version, etc.

**Reply by DmitryAndreyeff** [reply]
Hi @midichef , thanks! Opened an [issue](https://github.com/saulpw/visidata/issues/2634).
